stream-chat-android-offline/src/test/java/io/getstream/chat/android/offline/repository/facade/RepositoryFacadeIntegrationTests.kt

```diff
@@ -104,4 +111,27 @@ internal class RepositoryFacadeIntegrationTests : BaseRepositoryFacadeIntegratio
             Truth.assertThat(result!!.latestReactions).isEmpty()
             Truth.assertThat(result!!.ownReactions).isEmpty()
         }
+
```

Comment:
These 2 tests check that the assumption that when all embedded fields are null in the database, Room won't deserialize `ChannelInfoEntity` with all the inner nulls. It will set the whole reference (MessageInnerEntity.channelInfo) to null instead.